src/actions/counter.js

```diff
@@ -1,36 +0,0 @@
-const CounterTypes = require('action-types/counter');
```

Comment:
Let's delete the counter's tests as well.  That'll probably get the PR to pass travis, which is a nice warm feeling.